wiki-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -1,5 +1,180 @@
 <template>
   <v-app class="notesEditor">
-    <p>This is notes Management portlet!!</p>
+    <div>
+      <div
+        id="notesEditor"
+        class="notesEditor">
+        <div class="notesActions">
+          <div class="notesFormButtons">
+            <div class="notesFormLeftActions mr-10">
+              <img src="/wiki/images/wiki.png">
+              <input
+                id="notesTitle"
+                class="ml-4"
+                v-model="notes.title"
+                :maxlength="titleMaxLength"
+                :placeholder="notesTitlePlaceholder"
+                type="text">
+            </div>
+            <div class="notesFormRightActions">
+              <button
+                class="notesCancel btn mr-2"
+                @click="closeNotes">
+                {{ $t("btn.cancel") }}
+              </button>
+              <button
+                id="notesUpdateAndPost"
+                class="btn btn-primary"
+                @click="postNotes">
+                {{ $t("btn.post") }}
+              </button>
+            </div>
+          </div>
+        </div>
+        <div id="notesTop"></div>
+        <div class="formInputGroup">
+          <textarea
+            id="notesContent"
+            v-model="notes.body"
+            :placeholder="notesBodyPlaceholder"
+            class="notesFormInput"
+            name="notesContent">
+            </textarea>
+        </div>
+      </div>
+    </div>
   </v-app>
-</template>
\ No newline at end of file
+</template>
+
+<script>
+
+export default {
+  props: {
+  },
+  data() {
+    return {
+      notes: {
+        id: '',
+        title: '',
+        wikiType: 'portal',
+        wikiOwner: 'dw',
+        parentPageName: 'WikiHome',
```

Comment:
you can rename to NotesHome? if yes ,rename it please!

Reply:
No Don't rename the home wiki page name, we will need to upgrade the existing data before, so use WikiHome until we move to completely use the new app